node/test/node_test.cpp

```diff
@@ -95,12 +89,11 @@ namespace  bzn
     {
         this->node->start(nullptr);
 
-        this->yield();
         EXPECT_EQ(mock_io->socket_count, 1u);
 
         mock_io->do_incoming_connection(0);
         mock_io->ws_read_closures.at(0)(this->db_msg.SerializeAsString());
-        this->yield();
```

Comment:
ok replacing the "static" yield that simply waits for a time, with one that actually waits until things like outstanding callbacks are cleaned up.